projects/m2/019-morse-code/js/index.js

```diff
@@ -0,0 +1,77 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function convertToMorseCode(message, dictionary){

```

Comment:
Without the init function, how could I do the coding?
I think it's better to put the **dictionary inside the function** and pass as a parameter/argument only the message to encode.
It is always a good idea to make the function as autonomous and independent as possible.

Reply:
Ok, i put it into convertToMorseCode, thanks.